ecl/hql/hqlusage.cpp

```diff
@@ -416,8 +417,9 @@ IPropertyTree * SourceFieldUsage::createReport(bool includeFieldDetail, const IP
             return NULL;
     }
 
-    Owned<IPropertyTree> entry = createPTree(type);
+    Owned<IPropertyTree> entry = createPTree("datasource");
```

Comment:
This format change will need documenting - anyone who is currently reading the associated field information will need to know that this change is coming.